docs/CompatibleSoftware.rst

```diff
@@ -66,58 +66,121 @@ OCIO is natively integrated in
 Full support is provide for both image import/export, as well as image display.
 
 
+Katana
+******
 
-Nuke
-****
+`CG Pipeline / Lighting Tool - The Foundry <http://www.thefoundry.co.uk/products/katana>`__
 
-`Compositor - The Foundry <http://www.thefoundry.co.uk/products/nuke>`__
+Color management in Katana (all versions) natively relies on OCIO.
 
-Nuke 6.3v7+ ships with native support for OpenColorIO. The OCIO configuration
-is selectable in the user preferences.
+2D Nodes: OCIODisplay, OCIOColorSpace, OCIOCDLTransform
+Monitor Panel: Full OCIO Support
 
-OCIO Nodes: OCIOCDLTransform, OCIOColorSpace, OCIODisplay, OCIOFileTransform,
-OCIOLookConvert, OCIOLogConvert
 
-The OCIODisplay node is suitable for use in the Viewer as an input process (IP),
-and a register function is provides to add viewer options for each display upon
-launch.
+PhotoFlow
+*********
 
-The OCIO config "nuke-default" is provided, which matches the built-in Nuke
-color processing. This profile is useful for those who want to mirror the native
-nuke color processing in other applications.  (The underlying equations are
-also provided as python code in the config as well).
+`PhotoFlow <https://github.com/aferrero2707/PhotoFlow>`__
+ 
+It supports OCIO via a dedicated tool that can load a given configuration and apply the available color transforms. So far the tool has been tested it with the `Filmic <https://github.com/sobotka/filmic-blender>`__ and `ACES <https://opencolorio.org/configurations/aces_1.0.3.html>`__ configs.
 
-A `video demonstration <http://vimeo.com/38773736>`__ of the Nuke OCIO workflow.
 
-Mari
-****
+Unreal
+******
 
-`3D Paint - The Foundry <http://www.thefoundry.co.uk/products/mari>`__
+Unreal Engine got OCIO support in version 4.22, released in April 2019:
 
-Mari 1.4v1+ ships with native support for OpenColorIO in their display toolbar.
+Click for more details :
 
-A `video demonstration <http://vimeo.com/32909648>`__ of the Mari OCIO workflow.
+` Link 1 <https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_22/index.html>`__
 
-Katana
-******
+` Link 2 <https://docs.unrealengine.com/en-US/API/Plugins/OpenColorIO/index.html>`__
 
-`CG Pipeline / Lighting Tool - The Foundry <http://www.thefoundry.co.uk/products/katana>`__
 
-Color management in Katana (all versions) natively relies on OCIO.
+Mocha Pro 2020
+**************
 
-2D Nodes: OCIODisplay, OCIOColorSpace, OCIOCDLTransform
-Monitor Panel: Full OCIO Support
+Mocha Pro 2020 adds OCIO support:
 
-Hiero
-*****
+`Click for more details <https://borisfx.com/videos/opencolorio-mocha-pro-2020/>`__
 
-`Conform & Review - The Foundry <http://www.thefoundry.co.uk/products/hiero>`__
 
-Hiero 1.0 will ship with native support for OCIO in the display and the
-equivalent of Nuke's OCIOColorSpace in the Read nodes.
+Autodesk Arnold
```

Comment:
Each project should be consistently named. Some on the page include just the project name, and others also include the company/organization that make it. I would suggest this formatting for each:

`Project/Product (Company/Organization if there is one)`

So like:
```
Maya (Autodesk)
Nuke (Foundry)
```